src/main/scala/akka/contrib/datareplication/ORMultiMap.scala

```diff
@@ -7,21 +7,21 @@ import akka.cluster.{ UniqueAddress, Cluster }
 
 object ORMultiMap {
 
+  val _empty: ORMultiMap[Any] = new ORMultiMap(ORMap.empty)
   /**
    * Provides an empty multimap.
    */
-  val empty: ORMultiMap =
-    new ORMultiMap(ORMap.empty)
-  def apply(): ORMultiMap = empty
+  def empty[A]: ORMultiMap[A] = _empty.asInstanceOf[ORMultiMap[A]]
+  def apply(): ORMultiMap[Any] = _empty
 
   /**
    * Java API
    */
-  def create(): ORMultiMap = empty
+  def create[A](): ORMultiMap[A] = empty[A]
 
   def unapply(value: Any): Option[Map[String, Set[Any]]] = value match {
```

Comment:
note that the way to pattern match this without the extractor:

    def receive = {
      case Changed("key", data: ORMultiMap[Int] @unchecked) =>
        val x: Map[String, Set[Int]] = data.entries
    }

The unchecked is unfortunate, but I don't know a way around that.